README.md

```diff
@@ -13,33 +14,33 @@ Please note that the code, and especially the building and running story, is in
 a **pre-alpha** state. Please take a look, but don't be surprised if you hit
 bugs or missing pieces.
 
-## Building
 
-### In-place
+## Getting started
 
-To build the binaries in-place,
+To launch Scope inside of a Docker container,
 
 ```
-make build
+sudo wget -O /usr/local/bin/scope \
+  https://github.com/weaveworks/scope/releases/download/latest_release/scope
+sudo chmod a+x /usr/local/bin/scope
+sudo scope launch
```

Comment:
Have you tested this?  I hadn't got round to it yet (needed to have the images pushing to docker first).  That why I hadn't made a PR for them...